xbmc/music/dialogs/GUIDialogVisualisationPresetList.cpp

```diff
@@ -61,7 +62,8 @@ void CGUIDialogVisualisationPresetList::SetVisualisation(CVisualisation* vis)
   {
     SetUseDetails(false);
     SetMultiSelection(false);
-    SetHeading(StringUtils::Format(g_localizeStrings.Get(13407).c_str(), m_viz->Name().c_str()));
+    std::string heading = StringUtils::Format(g_localizeStrings.Get(13407).c_str(), m_viz->Name().c_str());
+    SetHeading(CVariant{heading});
```

Comment:
Both are fine really, when doing the original I mainly chose for readability so there's probably a mix.